### PR TITLE
Added calculator component

### PR DESCRIPTION
**Pull request by davestgermain**
[TNL-7138](https://openedx.atlassian.net/browse/TNL-7138)

calc2 2020-03-27 09_22_23

### CODE REVIEW

**Comment by davestgermain** [comment]
Calculator icon fixed to bottom of page:
Screen Shot 2020-03-20 at 1 28 52 PM

Calculator uncollapsed:
Screen Shot 2020-03-20 at 1 29 00 PM

Hint popover:
Screen Shot 2020-03-20 at 1 29 21 PM

Equation results:
Screen Shot 2020-03-20 at 1 30 15 PM

**Comment by davestgermain** [comment]
@marcotuts Could you sanity check this new UI? I know it's not perfect...

**Comment by marcotuts** [comment]
This might be an opportunity for us to move the calculator app visually into the sidebar format even if we aren't yet at a v1 of that extension boundary. My sense is that the time it would take to adjust current paragon base button / other patterns for this might be better spent moving this to the future location anyways. 

FYI @abutterworth  on this ^ - If we'd like to do this, perhaps we should merge this but flag it to be not visible for educators until we've done another separate pass at UI styling as a sidebar experience? thoughts on that?

File: src/courseware/course/calculator/index.jsx [hunk 1]
```diff
@@ -0,0 +1,257 @@
+import React, { Component } from 'react';
+import Button from 'react-bootstrap/Button';
+import Collapse from 'react-bootstrap/Collapse';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Popover from 'react-bootstrap/Popover';
```
Comment (abutterworth) [inline src/courseware/course/calculator/index.jsx]: Button and Collapse can be used from paragon: https://edx.github.io/paragon/components/collapsible

OverlayTrigger and Popover are not things we have in paragon yet... I wonder if we should add them now

File: src/courseware/course/calculator/index.jsx [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                </li>
+              </ul>
+            </li>
+            <li className="hint-item" id="hint-list">
+              <table className="table">
+                <thead>
+                  <tr>
+                    <th scope="col">To Use</th>
+                    <th scope="col">Type</th>
+                    <th scope="col">Examples</th>
```
Comment (abutterworth) [inline src/courseware/course/calculator/index.jsx]: I think we need translations here?

**Review by davidjoy — APPROVED** [review]

**Pull request merged**